content/sensu-go/5.15/api/federation.md

```diff
@@ -0,0 +1,372 @@
+http://127.0.0.1:8080/api/enterprise/federation/v1/etcd-replicators/my_replicator
+
+HTTP/1.1 204 No Content
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-delete-specification}
+
+/etcd-replicators/:etcd-replicator (DELETE) | 
+--------------------------|------
+description               | Deletes the specified replicator from Sensu.
+example url               | http://hostname:8080/api/enterprise/federation/v1/etcd-replicators/{etcd-replicator_name}
+response codes            | <ul><li>**Success**: 204 (No Content)</li><li>**Missing**: 404 (Not Found)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+
+## The `/cluster` endpoint
+
+### `/cluster` (GET)
+
+The `/cluster` endpoint provides HTTP GET access to a list of clusters.
+
+#### EXAMPLE {#cluster-get-example}
+
+The following example demonstrates a request to the `/cluster` endpoint, resulting in a list of clusters.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/enterprise/federation/v1/cluster -H "Authorization: Bearer $SENSU_TOKEN"
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "my_cluster"
+        },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+#### API Specification {#cluster-get-specification}
+
+/cluster (GET)  | 
+---------------|------
+description    | Returns a list of clusters.
+example url    | http://hostname:8080/api/enterprise/federation/v1/cluster
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "my_cluster"
+        },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+## The `/cluster/:cluster` endpoint {#the-clustercluster-endpoint}
+
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/cluster/:cluster` (PUT) {#clustercluster-put}
+
+#### EXAMPLE {#clustercluster-put-example}
+
+The following example demonstrates a request to update the cluster `my_cluster`.
+
+curl -X PUT \
+http://127.0.0.1:8080/api/enterprise/federation/v1/cluster/my_cluster -H "Authorization: Bearer $SENSU_TOKEN" \
+
+HTTP/1.1 200 OK
+{
+    "type": "Cluster",
+    "api_version": "federation/v1",
+    "metadata": {
+        "name": "my_cluster"
+    },
+    "spec": {
+        "api_urls": [
+            "http://10.0.0.1:8080",
+            "http://10.0.0.2:8080",
+            "http://10.0.0.3:8080"
+        ]
+    }
+}
```

Comment:
This should probably have shell highlighting around it.

Reply:
<img width="849" alt="Screen Shot 2019-10-22 at 1 54 13 PM" src="https://user-images.githubusercontent.com/10634078/67332076-76ce0200-f4d3-11e9-918d-d0e25ef82147.png">